<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AddPasswordDialog</class>
    <widget class="QDialog" name="AddPasswordDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>300</width>
                <height>200</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>添加密码</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QFormLayout" name="formLayout">
                    <item row="0" column="0">
                        <widget class="QLabel" name="label_website">
                            <property name="text">
                                <string>网站</string>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLineEdit" name="websiteEdit">
                            <property name="placeholderText">
                                <string>请输入网站地址</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="QLabel" name="label_username">
                            <property name="text">
                                <string>用户名</string>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="1">
                        <widget class="QLineEdit" name="usernameEdit">
                            <property name="placeholderText">
                                <string>请输入用户名</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="0">
                        <widget class="QLabel" name="label_password">
                            <property name="text">
                                <string>密码</string>
                            </property>
                        </widget>
                    </item>
                    <item row="2" column="1">
                        <widget class="QLineEdit" name="passwordEdit">
                            <property name="echoMode">
                                <enum>QLineEdit::Password</enum>
                            </property>
                            <property name="placeholderText">
                                <string>请输入密码</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections>
        <connection>
            <sender>buttonBox</sender>
            <signal>accepted()</signal>
            <receiver>AddPasswordDialog</receiver>
            <slot>accept()</slot>
        </connection>
        <connection>
            <sender>buttonBox</sender>
            <signal>rejected()</signal>
            <receiver>AddPasswordDialog</receiver>
            <slot>reject()</slot>
        </connection>
    </connections>
</ui> 